jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -156,7 +156,9 @@ private Thread(String vmName, Object vmThreadGroup, int vmPriority, boolean vmIs
 	super();
 
 	String threadName = (vmName == null) ? newName() : vmName;
+	/*[IF !Java15]*/
```

Comment:
Eventually we want to eliminate all the JavaNN flags in favour of using `JAVA_SPEC_VERSION`.
Rather than add new references to `Java15`, could you express this as:
```
/*[IF JAVA_SPEC_VERSION < 15]*/
```

Reply:
Updated accordingly.

Reply:
Thanks.